modules/social_features/social_out_of_office/social_out_of_office.info.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Social Out of office'
+description: 'Provides out of office functionality.'
+type: module
+core_version_requirement: ^9
+package: Social (experimental)
+lifecycle: experimental
+dependencies:
+  - social:social_profile
+  - social:social_post
```

Comment:
I would discard dependency on everything except `social_profile`, because technically it does not need post/comment/mentions but rather extends them. For example it extend private messages, but there is no dependency here

Reply:
@T2L  We are using a service from `social_mentions`. Isn't this a reason to have this dependency?

Reply:
Make sense. I've removed dependency.